<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="ADMIN-PC" name="com.crm.qa.testcases.LoginPageTestcases" tests="4" failures="0" timestamp="2020-10-11T00:33:22 IST" time="0.110" errors="1">
  <testcase name="loginTest" time="0.000" classname="com.crm.qa.testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- loginTest -->
  <system-out>
Test CaseloginTestis skipped Name of Test case is = null Status of Testcase is =3 Test CaseverifytitleTeststarted Name of Test case is = null Status of Testcase is =16  </system-out>
  <testcase name="Initalize" time="0.110" classname="com.crm.qa.testcases.LoginPageTestcases">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.crm.qa.base.BaseClass.Initalize(BaseClass.java:87)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:327)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Initalize -->
  <system-out/>
  <testcase name="imageTest" time="0.000" classname="com.crm.qa.testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- imageTest -->
  <system-out>
Test CaseimageTestis skipped Name of Test case is = null Status of Testcase is =3 Test CaseloginTeststarted Name of Test case is = null Status of Testcase is =16  </system-out>
  <testcase name="verifytitleTest" time="0.000" classname="com.crm.qa.testcases.LoginPageTestcases">
    <skipped/>
  </testcase> <!-- verifytitleTest -->
  <system-out>
Test CaseverifytitleTestis skipped Name of Test case is = null Status of Testcase is =3  </system-out>
</testsuite> <!-- com.crm.qa.testcases.LoginPageTestcases -->
